i18n/locale-en.json

```diff
@@ -0,0 +1,265 @@
+  "I18N_GALLERY_CATEGORIES_GOVERNMENT": "Government",
+  "I18N_GALLERY_CATEGORIES_HOBBIES": "Hobbies",
+  "I18N_GALLERY_CATEGORIES_INTERACTIVE_FICTION": "Interactive Fiction",
+  "I18N_GALLERY_CATEGORIES_LANGUAGES": "Languages",
+  "I18N_GALLERY_CATEGORIES_LAW": "Law",
+  "I18N_GALLERY_CATEGORIES_LIFE_SKILLS": "Life Skills",
+  "I18N_GALLERY_CATEGORIES_MATHS": "Maths",
+  "I18N_GALLERY_CATEGORIES_MATHEMATICS": "Mathematics",
+  "I18N_GALLERY_CATEGORIES_MEDICINE": "Medicine",
+  "I18N_GALLERY_CATEGORIES_MUSIC": "Music",
+  "I18N_GALLERY_CATEGORIES_PHILOSOPHY": "Philosophy",
+  "I18N_GALLERY_CATEGORIES_PHYSICS": "Physics",
+  "I18N_GALLERY_CATEGORIES_PROGRAMMING": "Programming",
+  "I18N_GALLERY_CATEGORIES_PSYCHOLOGY": "Psychology",
+  "I18N_GALLERY_CATEGORIES_PUZZLES": "Puzzles",
+  "I18N_GALLERY_CATEGORIES_READING": "Reading",
+  "I18N_GALLERY_CATEGORIES_RELIGION": "Religion",
+  "I18N_GALLERY_CATEGORIES_SPORT": "Sport",
+  "I18N_GALLERY_CATEGORIES_STATISTICS": "Statistics",
+  "I18N_GALLERY_CATEGORIES_TEST": "Test",
+  "I18N_GALLERY_CATEGORIES_WELCOME": "Welcome",
+  "I18N_GALLERY_N_CATEGORIES": "<[categoriesCount]> Categories",
+  "I18N_GALLERY_ALL_LANGUAGES": "All Languages",
+  "I18N_GALLERY_ALL_LANGUAGES_SELECTED": "All languages selected",
+  "I18N_GALLERY_LANGUAGES_EN": "English",
+  "I18N_GALLERY_LANGUAGES_AR": "العربية",
+  "I18N_GALLERY_LANGUAGES_BG": "български",
+  "I18N_GALLERY_LANGUAGES_CA": "català",
+  "I18N_GALLERY_LANGUAGES_ZH": "中文",
+  "I18N_GALLERY_LANGUAGES_HR": "hrvatski",
+  "I18N_GALLERY_LANGUAGES_CS": "čeština",
+  "I18N_GALLERY_LANGUAGES_DA": "dansk",
+  "I18N_GALLERY_LANGUAGES_NL": "Nederlands",
+  "I18N_GALLERY_LANGUAGES_TL": "Filipino",
+  "I18N_GALLERY_LANGUAGES_FI": "suomi",
+  "I18N_GALLERY_LANGUAGES_FR": "français",
+  "I18N_GALLERY_LANGUAGES_DE": "Deutsch",
+  "I18N_GALLERY_LANGUAGES_EL": "ελληνικά",
+  "I18N_GALLERY_LANGUAGES_HE": "עברית",
+  "I18N_GALLERY_LANGUAGES_HI": "हिन्दी",
+  "I18N_GALLERY_LANGUAGES_HU": "magyar",
+  "I18N_GALLERY_LANGUAGES_ID": "Bahasa Indonesia",
+  "I18N_GALLERY_LANGUAGES_IT": "italiano",
+  "I18N_GALLERY_LANGUAGES_JA": "日本語",
+  "I18N_GALLERY_LANGUAGES_KO": "한국어",
+  "I18N_GALLERY_LANGUAGES_LV": "latviešu",
+  "I18N_GALLERY_LANGUAGES_LT": "lietuvių",
+  "I18N_GALLERY_LANGUAGES_NO": "Norsk",
+  "I18N_GALLERY_LANGUAGES_FA": "فارسی",
+  "I18N_GALLERY_LANGUAGES_PL": "polski",
+  "I18N_GALLERY_LANGUAGES_PT": "português",
+  "I18N_GALLERY_LANGUAGES_RO": "română",
+  "I18N_GALLERY_LANGUAGES_RU": "русский",
+  "I18N_GALLERY_LANGUAGES_SR": "српски",
+  "I18N_GALLERY_LANGUAGES_SK": "slovenčina",
+  "I18N_GALLERY_LANGUAGES_SL": "slovenščina",
+  "I18N_GALLERY_LANGUAGES_ES": "español",
+  "I18N_GALLERY_LANGUAGES_SV": "svenska",
+  "I18N_GALLERY_LANGUAGES_TH": "ภาษาไทย",
+  "I18N_GALLERY_LANGUAGES_TR": "Türkçe",
+  "I18N_GALLERY_LANGUAGES_UK": "українська",
+  "I18N_GALLERY_LANGUAGES_VI": "Tiếng Việt",
+  "I18N_GALLERY_N_LANGUAGES": "<[languagesCount]> Languages",
```

Comment:
This needs plural support in many languages. See https://translatewiki.net/wiki/Plural for more details.